easycv/models/backbones/pytorch_image_models_wrapper.py

```diff
@@ -92,45 +91,32 @@ def __init__(self,
         if 'num_classes' not in kwargs:
             kwargs['num_classes'] = 0
 
+        if model_name in model_urls:
+            pretrained_path = checkpoint_path or model_urls[model_name]
+        else:
+            pretrained_path = checkpoint_path
+        print('load pretrained from {}'.format(pretrained_path))
         # create model by timm
         if model_name in timm_model_names:
-            try:
-                if pretrained and (model_name in model_urls):
-                    self.model = timm.create_model(model_name, False, '',
-                                                   scriptable, exportable,
-                                                   no_jit, **kwargs)
-                    self.init_weights(model_urls[model_name])
-                    print('Info: Load model from %s' % model_urls[model_name])
-
-                    if checkpoint_path is not None:
-                        self.init_weights(checkpoint_path)
+            self.model = timm.create_model(model_name, False, '', scriptable,
+                                           exportable, no_jit, **kwargs)
+
+            if pretrained:
```

Comment:
not to delete the `init_weights` api, and move the code pieces for loading the pretrained model to `init_weights`